galaxy.yml

```diff
@@ -2,14 +2,15 @@
 
 namespace: community
 name: molecule
-version: 0.1.0
```

Comment:
Does this mean that this collection is now released as v1.0.0 including semver guarantees?